core/utils.py

```diff
@@ -0,0 +1,24 @@
+import jwt
+
+from django.http import JsonResponse
+from django.conf import settings
+
+from users.models import User
+
+def LoginConfirm(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            token        = request.headers.get('Authorization', None)
+            payload      = jwt.decode(token, settings.SECRET_KEY, settings.ALGORITHM)
+            user         = User.objects.get(id=payload['id'])
+            request.user = user
+        
+        except jwt.exceptions.DecodeError:
+            return JsonResponse({'message' : 'INVALID_TOKEN' }, status=400)
+
+        except User.DoesNotExist:
+            return JsonResponse({'message' : 'INVALID_USER'}, status=400)
```

Comment:
인증관련 status code는 401을 사용합니다.
```suggestion
        except jwt.exceptions.DecodeError:
            return JsonResponse({'message' : 'INVALID_TOKEN' }, status=401)

        except User.DoesNotExist:
            return JsonResponse({'message' : 'INVALID_USER'}, status=401)
```

Reply:
수정완료했습니다!